### DISCUSSION TITLE
NFT collection marked as spam

### DISCUSSION THREAD

**Discussion by cesar4design** [phantom/docs#279] [Help & Support] [answered]
### Summary

I was launching a collection using metaplex/candymachine standard for a friend, I minted the first NFT and it got marked as spam. So I re-launched the collection and it got flagged as spam. I would like you to review it as it is not spam. It is only 60 legitimate NFTs that I am going to mint to distribute to our token holders.

The NFT is https://solscan.io/token/FFQRtJTtbJZ74C6yP7guymAApg9LD12uDrduYHdXs85k

We are a legitimate project, twitter: @dkenonsol, site: https://deadkennycoin.rip/

### Steps to Reproduce

Mint a NFT using Candy Machine

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by cesar4design** [comment]
I don't understand what the "spam" is in a collection of currently 1 NFT (with max mint 60 NFT), and that has been minted directly to the same wallet that launched the program. Can you help me please @adamdelphantom ?

**Reply by adamdelphantom** [reply, marked as answer]
@cesar4design Should be good to go.

**Reply by SolVibe** [reply]
@adamdelphantom 
Please check this collection too.
https://solscan.io/collection/2914b6ffe60b5f49ec46b5ea5978739da5aae58ae148c54f5d70b98bf23946e2

**Reply by adamdelphantom** [reply]
@reza2870 can you provide more information about the collection, please?

**Reply by SolVibe** [reply]
@adamdelphantom 
Yes. This is from Dropzone dapp.
A dapp for farming of airdrops.

X profile
https://x.com/bend_capital

Discord:
https://discord.com/invite/hVurZZx5
